Makefile

```
@@ -1,8 +1,11 @@
+PINT_VERSION='0.42.2'
+
 .PHONY: all
-all: prepare lint test_rules
+all: prepare lint test_rules pint_lint
 
 .PHONY: prepare
 prepare:
+	make pint
```

I think it's better to have pint as a dependency to prepare rather than invoking `make` again.
```suggestion
prepare: pint
```